File: src/main/java/org/dedda/bratwurst/parse/ReadLineParser.java
package org.dedda.bratwurst.parse;

import org.dedda.bratwurst.lang.io.terminal.ReadLine;

/**
 * Created by dedda on 1/28/16.
 *
 * @author dedda
 */
public class ReadLineParser extends ExpressionParser {

    @Override
    public ReadLine parse(String data, int lineNumber) {
        return new ReadLine(lineNumber);
    }
}

File: src/main/java/org/dedda/bratwurst/parse/ReadLineParser.kt
package org.dedda.bratwurst.parse

import org.dedda.bratwurst.lang.io.terminal.ReadLine

/**
 * Created by dedda on 1/28/16.
 *
 * @author dedda
 */
class ReadLineParser : ExpressionParser() {
    override fun parse(data: String, lineNumber: Int): ReadLine {
        return ReadLine(lineNumber)
    }
}